CustomerJourney/src/wrapper.ts

```diff
@@ -0,0 +1,25 @@
+import { html, LitElement } from "lit-element";
+import { customElementWithCheck } from "./mixins/CustomElementCheck";
+import "@momentum-ui/web-components/dist/comp/md-theme";
+import "@cjaas/common-components";
+import styles from "./assets/styles/wrapper.scss";
+
+@customElementWithCheck("customer-journey-widget-wrapper")
```

Comment:
1. try to keep the name of the file more similar to `@customElementWithCheck("customer-journey-widget-wrapper")`

2. Looks like this is just a md-theme wrapper. maybe incorporate that into the file name and component name. It is more descripive.

Reply:
Do we need to handle light and dark mode here?

Reply:
Updated.

This is meant for CCE and CCX. Not sure dark mode plays any part there. Upon request it can be added.